test/cmd/sdn.sh

```diff
@@ -4,29 +4,82 @@ trap os::test::junit::reconcile_output EXIT
 
 os::test::junit::declare_suite_start "cmd/sdn"
 
+# ClusterNetworks
 os::cmd::expect_success 'oc get clusternetworks'
 os::cmd::expect_success_and_text 'oc get clusternetwork default -o jsonpath="{.pluginName}"' 'redhat/openshift-ovs-multitenant'
 os::cmd::expect_failure_and_text 'oc patch clusternetwork default -p "{\"network\": \"1.0.0.0/8\"}"' 'Invalid value'
 os::cmd::expect_failure_and_text 'oc patch clusternetwork default -p "{\"hostsubnetlength\": 22}"' 'Invalid value'
 os::cmd::expect_failure_and_text 'oc patch clusternetwork default -p "{\"serviceNetwork\": \"1.0.0.0/8\"}"' 'Invalid value'
 
+# NetNamespaces
 orig_project="$(oc project -q)"
 
 os::cmd::expect_success 'oc get netnamespaces'
 os::cmd::expect_success_and_text 'oc get netnamespace default -o jsonpath="{.netid}"' '^0$'
-os::cmd::expect_failure 'oc get netnamespace sdn-test'
-os::cmd::expect_success 'oc new-project sdn-test'
-os::cmd::expect_success 'oc get namespace sdn-test'
-os::cmd::try_until_success 'oc get netnamespace sdn-test'
-os::cmd::expect_success_and_not_text 'oc get netnamespace sdn-test -o jsonpath="{.netid}"' '^0$'
-os::cmd::expect_success 'oc delete namespace sdn-test'
-os::cmd::try_until_failure 'oc get netnamespace sdn-test'
+
+os::cmd::expect_failure 'oc get netnamespace sdn-test-1'
```

Comment:
We don't have a habit of checking assumptions like this. Given a good cleanup script for advanced use-cases for the tests, we don't need it. In a normal `test-cmd` run, we definitely don't need it.